big_tests/tests/gdpr_SUITE.erl

```diff
@@ -194,6 +249,26 @@ inbox_opts() ->
      {groupchat, [muclight]},
      {markers, [displayed]}].
 
+inbox_opts(muc) ->
+    [{aff_changes, true},
+     {remove_on_kicked, true},
+     {groupchat, [muc]},
+     {markers, [displayed]}].
```

Comment:
Why cannot we have one definition:
```
inbox_opts(MUCType) ->
    [{aff_changes, true},
     {remove_on_kicked, true},
     {groupchat, [MUCType]},
     {markers, [displayed]}].
```
with `MUCType` (or smth like this) as `muc` or `muclight` ?